src/pages/LikePage/LikePage.jsx

```diff
@@ -0,0 +1,24 @@
+import { Sidebar, LikeCard } from "../../components/index";
+import {useLike} from "../../context/LikeContext"
+
+export function LikePage() {
+
+    const {likeState: {likedVideo}} = useLike()
+    // console.log(likedVideo)
```

Comment:
Remove the comments. The code structure is good, just need to work on the UI of the like button